finsoc/app/assets/stylesheets/newsletter.scss

```diff
@@ -3,4 +3,48 @@
 		width: 100%;
 		height: 500px;
 	}
-}
\ No newline at end of file
+}
+/* Dropdown Button */
+.dropbtn {
+    background-color: #4CAF50;
+    color: white;
+    padding: 16px;
+    font-size: 16px;
+    border: none;
+    cursor: pointer;
+
+}
+
+/* Dropdown button on hover & focus */
+.dropbtn:hover, .dropbtn:focus {
+    background-color: #3e8e41;
+}
+
+/* The container <div> - needed to position the dropdown content */
+.dropdown {
```

Comment:
The dropdown covers the title of the pdf in the viewer for some browsers. Might be better to place the dropdown above the viewer instead.